.github/workflows/python.yml

```diff
@@ -141,8 +141,8 @@ jobs:
             - name: Build Python wrapper
               uses: ./.github/workflows/build-python-wrapper
               with:
-                  os: "macos-latest"
-                  target: "x86_64-apple-darwin"
+                  os: "macos"
+                  target: "aarch64-apple-darwin"
```

Comment:
Why we do it just for aarch and not both arch and x86?

Reply:
Same question for ubuntu

Reply:
+1 for mac

Re: ubuntu on arm CPU - we never tried it before. Worth adding in scope of another PR.

Reply:
@avifenesh out of scope of this PR - can be added in a different one